.gitignore

```diff
@@ -1,6 +1,7 @@
 *.pyc
 __pycache__
 hairenv
```

Comment:
I think this needs to be `myenv/` since its a dir

Reply:
updated .gitignore as requested merge my branch